### DISCUSSION TITLE
Flatpak apps are not listed in my newly installed PopOS

### DISCUSSION THREAD

**Discussion by GhanashyamGhimire** [pop-os/pop#1920] [Q&A] [answered]
Hello,

I'm new here, so please forgive me if I posted this to a wrong channel.

I started using PopOS myself from a few weeks ago and so far no regrets of my departure from Windows.
My friend had a laptop that kept freezing all the time with unlicensed Windows 10, so I suggested him to try PopOS for a few weeks to see if he'll like it and can stick to it, or he can always go back to Windows if it didn't suit him.

So I swapped out to a new SSD on his laptop and installed PopOS on there and the machine is running blazing fast now.
But I noticed that my Pop_Shop! is not showing any Flatpak apps. I tried to Google this and follow many recommendations from PopOS main page as well as other suggestions from forum users to uninstall and reinstall back but none seems to work.

Could anyone suggest what to do? :)

Thank you in advance!

Best regards,
Shyam

![Screenshot from 2021-09-21 13-06-08](https://user-images.githubusercontent.com/91110498/134114480-cf49f29f-441c-45b4-8957-ff127d9724d6.png)

**Comment by GhanashyamGhimire** [comment, marked as answer]
It looks like this issue was self resolved after a few hours.
I applied the fixes suggested in forums and did several restarts before reporting this issue on here, but when I check it again here after a few hours, I am able to see those Flatpak apps now.

**Reply by mmstick** [reply]
The appcenter is rather slow to update when it has nothing cached at all on a fresh install.
